index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+
+<!-- jquery library -->
+
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+
+  <!-- Bootstrap stylesheet -->
+
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
+
+  <!-- extra style -->
+  <link rel="stylesheet" type="text/css" href="assets/style.css">
+  <!-- Firebase  -->
+
+<script src="https://www.gstatic.com/firebasejs/4.6.0/firebase.js"></script>
+<script>
+  // Initialize Firebase
+  var config = {
+    apiKey: "AIzaSyByWTmqM7_8TfNxDzGe5dskTjYl5R84NdA",
+    authDomain: "train-schedule-d6927.firebaseapp.com",
+    databaseURL: "https://train-schedule-d6927.firebaseio.com",
+    projectId: "train-schedule-d6927",
+    storageBucket: "train-schedule-d6927.appspot.com",
+    messagingSenderId: "132794670641"
+  };
+  firebase.initializeApp(config);
+</script>
```

Comment:
I don't think this should be on the html page, it would make more sense to just have this in your js file. It is good to start practicing separation of concerns, meaning making code that is relevant or dependant on itself contained in a way that makes sense.